recipes/pngpack-py/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: pngpack-py
+  version: {{ version }}
+
+source:
+  url: https://github.com/axiom-data-science/pngpack/archive/v{{ version }}.zip
+  sha256: df6c6011395fb36add7c184cfbea832aef32fcee21069dec63dd27842a36ab03
+
+build:
+  number: 0
+  skip: true  # [py<35 or win]
+  script: cd pngpack-py && {{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
This is fine. Once the library is available on PyPI or something like it we can revisit the pip install. (I'll take a look at your setup.py too later today.)

Reply:
I don't believe this will be going up to PyPI, unless someone else can assist. I could not get it working due to dynamic linking to libpng, even using the wheel audit tool.

Reply:
I can probably help you with that but you can upload a source distribution only. No need to worry about wheels at this moment.